qsirecon/tests/data/scalar_mapper_outputs.txt

```diff
@@ -10,24 +10,74 @@ derivatives/qsirecon-DIPYDKI/logs/CITATION.tex
 derivatives/qsirecon-DIPYDKI/sub-ABCD
 derivatives/qsirecon-DIPYDKI/sub-ABCD.html
 derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi
+derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-MNI152NLin2009cAsym_model-GQI_param-gfa_dwimap.json
+derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-MNI152NLin2009cAsym_model-GQI_param-gfa_dwimap.nii.gz
+derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-MNI152NLin2009cAsym_model-GQI_param-iso_dwimap.json
+derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-MNI152NLin2009cAsym_model-GQI_param-iso_dwimap.nii.gz
+derivatives/qsirecon-DIPYDKI/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-MNI152NLin2009cAsym_model-GQI_param-qa_dwimap.json
```

Comment:
these should be in the dsistudio qsirecon suffix

Reply:
Thanks for catching that. I'll look into it.

Reply:
I think `scalar_gatherer` is the culprit

Reply:
ah! we might not need scalar_gatherer at all now that the scalar mappers are being duplicated for each mapping

Reply:
Fixed!